contrib/QAT-ZSTD-Plugin-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+# Intel® QuickAssist Technology ZSTD Plugin (QAT ZSTD Plugin) is a plugin to Zstandard*(ZSTD*) for accelerating compression by QAT.
+
+if (ENABLE_QAT_OUTOFTREE)
```

Comment:
Cosmetic: Please intent in CMake if/else blocks

Reply:
Done